apps/daimo-mobile/src/view/screen/onboarding/OnboardingChooseNameScreen.tsx

```diff
@@ -79,12 +79,30 @@ export function OnboardingPickNameScreen({ route }: Props) {
   );
 }
 
-function IconRow(props: { icon: OctName; color?: string; title: string }) {
+function CoverImage() {
+  return (
+    <View style={{ display: "flex", alignItems: "center" }}>
+      <Image source={ImgChooseName} style={{ width: 188, height: 156 }} />
```

Comment:
we have a similar `CoverGraphic` as well, might be able to cleanly merge?

Reply:
fixed